sdk/aqueduct/responses.py

```diff
@@ -22,8 +43,10 @@ class OperatorResult(BaseModel):
             Only set if this operator represents a unit test on an artifact.
     """
 
-    logs: Optional[Dict[str, str]]
-    err_msg: Optional[str]
+    user_logs: Optional[Logs] = None
+    error: Optional[Error] = None
+    status: ExecutionStatus = ExecutionStatus.UNKNOWN
+    failure_type: FailureType = FailureType.SUCCESS
```

Comment:
why do these fields need default values? Aren't they only fetched from the backend? The backend should supply all the appropriate values, and if they sometimes don't, that field should be Optional.

Reply:
I think it makes sense to put `FailureType` optional as we discussed in #144 . I think it make sense to default `status` to `Unknown` which is more explicit than `status = None`